_sass/_skeleton_custom.scss

```diff
@@ -37,6 +38,7 @@
   .has-docked-nav .navbar-spacer { display : block; }
   /* Re-overiding the width 100% declaration to match size of % based container */
   .has-docked-nav .navbar > .container { width : 80%; }
+  .visible-xs { display : none; }
```

Comment:
This probably should be refined to say otherwise hidden at larger resolutions... we're using Skeleton instead of Bootstrap, but I used the BS class names.